src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -20,6 +22,10 @@
     <ProjectJson>unix/project.json</ProjectJson>
     <ProjectLockJson>unix/project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'net46' and '$(ProjectJson)' == '' ">
+    <ProjectJson>net46/project.json</ProjectJson>
```

Comment:
Any particular reason you need a separate project.json? for net46 you should be able to add a section for it in the default project.json.